[I 20190506 16:54:03 basehttp:124] "GET /index/ HTTP/1.1" 200 13294
[W 20190506 16:54:04 base:93] Not Found: /favicon.ico
[W 20190506 16:54:04 basehttp:124] "GET /favicon.ico HTTP/1.1" 404 3372
[W 20190506 16:54:19 basehttp:124] "GET /static/vendor/bootstrap/js/bootstrap.min.js.map HTTP/1.1" 404 1744
[I 20190506 16:55:25 basehttp:124] "GET /index/ HTTP/1.1" 200 13294
[W 20190506 16:55:25 basehttp:124] "GET /static/vendor/bootstrap/js/bootstrap.min.js.map HTTP/1.1" 404 1744
[I 20190506 16:57:33 basehttp:124] "GET /index/ HTTP/1.1" 200 13294
[W 20190506 16:57:34 base:93] Not Found: /favicon.ico
[W 20190506 16:57:34 basehttp:124] "GET /favicon.ico HTTP/1.1" 404 3372
[I 20190506 17:04:07 basehttp:124] "GET /userlist HTTP/1.1" 301 0
[I 20190506 17:04:07 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10407
[I 20190506 17:10:55 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10394
[I 20190506 17:10:58 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10394
[I 20190506 17:11:48 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10393
[I 20190506 17:13:46 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10392
[I 20190506 17:13:48 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10392
[I 20190506 17:19:14 basehttp:124] "GET /index/ HTTP/1.1" 200 13293
[I 20190506 21:45:09 basehttp:124] "GET /killttyp HTTP/1.1" 301 0
[I 20190506 21:45:09 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 10892
[I 20190506 21:45:09 basehttp:124] "GET /static/vendor/simple-line-icons/css/simple-line-icons.css HTTP/1.1" 304 0
[I 20190506 21:45:09 basehttp:124] "GET /static/vendor/font-awesome/css/fontawesome-all.min.css HTTP/1.1" 304 0
[I 20190506 21:45:09 basehttp:124] "GET /static/css/styles.css HTTP/1.1" 304 0
[I 20190506 21:45:09 basehttp:124] "GET /static/vendor/jquery/jquery.min.js HTTP/1.1" 304 0
[I 20190506 21:45:09 basehttp:124] "GET /static/vendor/popper.js/popper.min.js HTTP/1.1" 304 0
[I 20190506 21:45:09 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 200 3731
[I 20190506 21:45:09 basehttp:124] "GET /static/vendor/bootstrap/js/bootstrap.min.js HTTP/1.1" 304 0
[I 20190506 21:45:09 basehttp:124] "GET /static/vendor/chart.js/chart.min.js HTTP/1.1" 304 0
[I 20190506 21:45:09 basehttp:124] "GET /static/js/carbon.js HTTP/1.1" 304 0
[I 20190506 21:45:09 basehttp:124] "GET /static/js/demo.js HTTP/1.1" 304 0
[I 20190506 21:45:10 basehttp:124] "GET /static/vendor/font-awesome/webfonts/fa-solid-900.woff2 HTTP/1.1" 304 0
[I 20190506 21:45:10 basehttp:124] "GET /static/vendor/simple-line-icons/fonts/Simple-Line-Icons.woff2?v=2.4.0 HTTP/1.1" 304 0
[I 20190506 21:45:10 basehttp:124] "GET /static/vendor/font-awesome/webfonts/fa-solid-900.woff HTTP/1.1" 304 0
[I 20190506 21:45:10 basehttp:124] "GET /static/vendor/simple-line-icons/fonts/Simple-Line-Icons.ttf?v=2.4.0 HTTP/1.1" 304 0
[W 20190506 21:45:10 base:93] Not Found: /favicon.ico
[W 20190506 21:45:10 basehttp:124] "GET /favicon.ico HTTP/1.1" 404 3498
[I 20190506 21:45:14 basehttp:124] "GET /index/ HTTP/1.1" 200 12422
[I 20190506 21:45:22 basehttp:124] "GET /userlist/ HTTP/1.1" 200 7867
[I 20190506 21:45:26 basehttp:124] "GET /userlist/ HTTP/1.1" 200 7867
[I 20190506 21:45:33 basehttp:124] "GET /login/ HTTP/1.1" 200 2951
[I 20190506 21:46:20 basehttp:124] "POST /login/ HTTP/1.1" 200 13293
[I 20190506 21:46:20 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[I 20190506 21:46:20 basehttp:124] "GET /static/imgs/avatar-1.png HTTP/1.1" 304 0
[I 20190506 21:46:23 basehttp:124] "GET /userlist/ HTTP/1.1" 200 9564
[I 20190506 21:46:52 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11763
[I 20190506 21:46:56 basehttp:124] "GET /admin HTTP/1.1" 301 0
[I 20190506 21:46:56 basehttp:124] "GET /admin/ HTTP/1.1" 200 3912
[I 20190506 21:46:56 basehttp:124] "GET /static/admin/css/base.css HTTP/1.1" 200 16106
[I 20190506 21:46:56 basehttp:124] "GET /static/admin/css/dashboard.css HTTP/1.1" 200 412
[I 20190506 21:46:56 basehttp:124] "GET /static/admin/css/responsive.css HTTP/1.1" 200 17894
[I 20190506 21:46:56 basehttp:124] "GET /static/admin/css/fonts.css HTTP/1.1" 200 423
[I 20190506 21:46:57 basehttp:124] "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 81348
[I 20190506 21:46:57 basehttp:124] "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 200 82564
[I 20190506 21:46:57 basehttp:124] "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
[I 20190506 21:46:57 basehttp:124] "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 80304
[I 20190506 21:46:57 basehttp:124] "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
[I 20190506 21:47:00 basehttp:124] "GET /admin/users/userprofile/add/ HTTP/1.1" 200 14133
[I 20190506 21:47:00 basehttp:124] "GET /static/admin/css/forms.css HTTP/1.1" 200 8173
[I 20190506 21:47:00 basehttp:124] "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 363
[I 20190506 21:47:00 basehttp:124] "GET /static/admin/js/core.js HTTP/1.1" 200 7134
[I 20190506 21:47:00 basehttp:124] "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 6897
[I 20190506 21:47:00 basehttp:124] "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 258648
[I 20190506 21:47:00 basehttp:124] "GET /static/admin/js/actions.js HTTP/1.1" 200 6502
[I 20190506 21:47:00 basehttp:124] "GET /admin/jsi18n/ HTTP/1.1" 200 7684
[I 20190506 21:47:00 basehttp:124] "GET /static/admin/css/widgets.css HTTP/1.1" 200 10340
[I 20190506 21:47:00 basehttp:124] "GET /static/admin/js/urlify.js HTTP/1.1" 200 8729
[I 20190506 21:47:00 basehttp:124] "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1538
[I 20190506 21:47:00 basehttp:124] "GET /static/admin/js/calendar.js HTTP/1.1" 200 7777
[I 20190506 21:47:00 basehttp:124] "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 128820
[I 20190506 21:47:00 basehttp:124] "GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1" 200 20283
[I 20190506 21:47:00 basehttp:124] "GET /static/admin/js/change_form.js HTTP/1.1" 200 712
[I 20190506 21:47:00 basehttp:124] "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 495
[I 20190506 21:47:01 basehttp:124] "GET /static/admin/img/icon-calendar.svg HTTP/1.1" 200 1086
[I 20190506 21:47:01 basehttp:124] "GET /static/admin/img/icon-clock.svg HTTP/1.1" 200 677
[I 20190506 21:47:32 basehttp:124] "POST /admin/users/userprofile/add/ HTTP/1.1" 302 0
[I 20190506 21:47:32 basehttp:124] "GET /admin/users/userprofile/ HTTP/1.1" 200 4837
[I 20190506 21:47:32 basehttp:124] "GET /admin/jsi18n/ HTTP/1.1" 200 7684
[I 20190506 21:47:32 basehttp:124] "GET /static/admin/css/changelists.css HTTP/1.1" 200 6170
[I 20190506 21:47:32 basehttp:124] "GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
[I 20190506 21:47:32 basehttp:124] "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
[I 20190506 21:47:39 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10404
[I 20190506 21:47:39 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[I 20190506 21:47:49 basehttp:124] "GET /admin/users/ HTTP/1.1" 200 2423
[I 20190506 21:47:51 basehttp:124] "GET /admin/users/userprofile/ HTTP/1.1" 200 4662
[I 20190506 21:47:51 basehttp:124] "GET /admin/jsi18n/ HTTP/1.1" 200 7684
[I 20190506 21:47:53 basehttp:124] "GET /admin/users/userprofile/1/change/ HTTP/1.1" 200 14510
[I 20190506 21:47:53 basehttp:124] "GET /admin/jsi18n/ HTTP/1.1" 200 7684
[I 20190506 21:48:00 basehttp:124] "POST /admin/users/userprofile/1/change/ HTTP/1.1" 302 0
[I 20190506 21:48:00 basehttp:124] "GET /admin/users/userprofile/ HTTP/1.1" 200 4835
[I 20190506 21:48:00 basehttp:124] "GET /admin/jsi18n/ HTTP/1.1" 200 7684
[I 20190506 21:48:03 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10419
[I 20190506 21:48:49 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11763
[I 20190506 21:48:49 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[W 20190506 21:48:52 base:93] Not Found: /killttyp/tables.html
[W 20190506 21:48:52 basehttp:124] "GET /killttyp/tables.html HTTP/1.1" 404 3525
[I 20190506 21:48:55 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 11754
[I 20190506 21:49:02 basehttp:124] "GET /addhost/ HTTP/1.1" 200 13949
[I 20190506 21:49:17 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10419
[I 20190506 21:52:36 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11247
[I 20190506 21:52:36 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[I 20190506 21:52:58 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11233
[I 20190506 21:53:13 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11227
[I 20190506 21:53:40 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11255
[I 20190506 21:54:22 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11245
[I 20190506 21:54:22 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[I 20190506 21:54:43 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11241
[I 20190506 21:55:01 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11263
[I 20190506 21:55:09 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11267
[I 20190506 21:55:31 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11279
[I 20190506 21:55:48 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10441
[I 20190506 21:56:04 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11281
[I 20190506 21:56:17 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10287
[I 20190506 21:56:17 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[I 20190506 21:56:35 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11087
[I 20190506 21:56:44 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11927
[I 20190506 21:56:57 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11127
[I 20190506 21:57:49 basehttp:124] "GET /userlist/ HTTP/1.1" 200 12209
[I 20190506 21:58:03 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11371
[I 20190506 21:58:21 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11371
[I 20190506 21:58:21 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[I 20190506 21:59:21 basehttp:124] "GET /userlist/ HTTP/1.1" 200 12375
[I 20190506 22:00:31 basehttp:124] "GET /userlist/ HTTP/1.1" 200 13792
[I 20190506 22:00:31 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[I 20190506 22:01:02 basehttp:124] "GET /userlist/ HTTP/1.1" 200 13792
[I 20190506 22:01:06 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11139
[I 20190506 22:01:42 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11137
[I 20190506 22:01:47 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11137
[I 20190506 22:02:07 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11267
[I 20190506 22:02:31 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11139
[I 20190506 22:04:15 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11347
[I 20190506 22:04:15 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[W 20190506 22:04:15 base:93] Not Found: /userlist/imgs/avatar-1.png
[W 20190506 22:04:15 basehttp:124] "GET /userlist/imgs/avatar-1.png HTTP/1.1" 404 3543
[I 20190506 22:04:29 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11357
[W 20190506 22:04:29 base:93] Not Found: /userlist/static/imgs/avatar-1.png
[W 20190506 22:04:29 basehttp:124] "GET /userlist/static/imgs/avatar-1.png HTTP/1.1" 404 3564
[I 20190506 22:04:54 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11359
[I 20190506 22:05:12 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11353
[I 20190506 22:05:37 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11291
[I 20190506 22:05:59 basehttp:124] "GET /userlist/ HTTP/1.1" 200 11293
[I 20190506 22:06:10 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10955
[I 20190506 22:06:29 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10963
[I 20190506 22:06:38 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10963
[I 20190506 22:06:44 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10963
[I 20190506 22:06:59 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10929
[I 20190506 22:06:59 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[I 20190506 22:07:07 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10953
[I 20190506 22:16:15 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10966
[I 20190506 22:16:15 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[I 20190506 22:16:18 basehttp:124] "GET /usersettings/ HTTP/1.1" 200 13901
[I 20190506 22:16:27 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10966
[I 20190506 22:27:18 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10967
[I 20190506 22:27:18 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[I 20190506 22:27:41 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10969
[I 20190506 22:28:53 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10971
[I 20190506 22:28:54 basehttp:124] "GET /index/ HTTP/1.1" 200 13311
[I 20190506 22:29:36 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10971
[W 20190506 22:29:37 base:93] Not Found: /userlist/tables.html
[W 20190506 22:29:37 basehttp:124] "GET /userlist/tables.html HTTP/1.1" 404 3659
[I 20190506 22:29:49 basehttp:124] "GET /index/ HTTP/1.1" 200 13311
[I 20190506 22:30:16 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 11772
[I 20190506 22:30:21 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11781
[W 20190506 22:31:17 base:93] Not Found: /killttyp/500.html
[W 20190506 22:31:17 basehttp:124] "GET /killttyp/500.html HTTP/1.1" 404 3650
[I 20190506 22:31:23 basehttp:124] "GET /index/ HTTP/1.1" 200 13311
[I 20190506 22:31:43 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 11772
[I 20190506 22:31:44 basehttp:124] "GET /addhost/ HTTP/1.1" 200 13967
[I 20190506 22:31:56 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 10442
[I 20190506 22:31:57 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190506 22:32:07 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 10442
[I 20190506 22:32:08 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190506 22:32:19 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 10442
[I 20190506 22:32:21 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190506 22:32:22 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[I 20190506 22:32:24 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 12802
[I 20190506 22:32:26 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190506 22:32:46 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 10442
[I 20190506 22:32:53 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 13979
[I 20190506 22:32:59 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 12249
[I 20190506 22:33:04 basehttp:124] "GET /addhost/ HTTP/1.1" 200 14477
[I 20190506 22:33:53 basehttp:124] "POST /collecthost/ HTTP/1.1" 200 14619
[I 20190506 22:34:16 basehttp:124] "POST /collecthost/ HTTP/1.1" 200 14619
[I 20190506 22:34:33 basehttp:124] "POST /addhost/ HTTP/1.1" 200 13967
[E 20190506 22:34:35 exception:118] Internal Server Error: /hostinfo/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 71, in execute
    return self.cursor.execute(query, args)
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/cursors.py", line 255, in execute
    self.errorhandler(self, exc, value)
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/connections.py", line 50, in defaulterrorhandler
    raise errorvalue
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/cursors.py", line 252, in execute
    res = self._query(query)
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/cursors.py", line 378, in _query
    db.query(q)
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/connections.py", line 280, in query
    _mysql.connection.query(self, query)
_mysql_exceptions.ProgrammingError: (1146, "Table 'opsauto.hostinfo_host_group' doesn't exist")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/hostsinfo/views.py", line 14, in get
    return render(request,"hosts-list.html",{'hosts':hosts,'groups':groups})
  File "/opt/python3.6/lib/python3.6/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/loader_tags.py", line 155, in render
    return compiled_parent._render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/loader_tags.py", line 67, in render
    result = block.nodelist.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/defaulttags.py", line 211, in render
    nodelist.append(node.render_annotated(context))
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/defaulttags.py", line 168, in render
    len_values = len(values)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/query.py", line 254, in __len__
    self._fetch_all()
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/query.py", line 1179, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/query.py", line 53, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1067, in execute_sql
    cursor.execute(sql, params)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/backends/utils.py", line 100, in execute
    return super().execute(sql, params)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/backends/utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/backends/utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 71, in execute
    return self.cursor.execute(query, args)
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/cursors.py", line 255, in execute
    self.errorhandler(self, exc, value)
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/connections.py", line 50, in defaulterrorhandler
    raise errorvalue
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/cursors.py", line 252, in execute
    res = self._query(query)
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/cursors.py", line 378, in _query
    db.query(q)
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/connections.py", line 280, in query
    _mysql.connection.query(self, query)
django.db.utils.ProgrammingError: (1146, "Table 'opsauto.hostinfo_host_group' doesn't exist")
[E 20190506 22:34:36 basehttp:124] "GET /hostinfo/ HTTP/1.1" 500 280944
[E 20190506 22:34:48 exception:118] Internal Server Error: /hostinfo/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 71, in execute
    return self.cursor.execute(query, args)
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/cursors.py", line 255, in execute
    self.errorhandler(self, exc, value)
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/connections.py", line 50, in defaulterrorhandler
    raise errorvalue
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/cursors.py", line 252, in execute
    res = self._query(query)
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/cursors.py", line 378, in _query
    db.query(q)
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/connections.py", line 280, in query
    _mysql.connection.query(self, query)
_mysql_exceptions.ProgrammingError: (1146, "Table 'opsauto.hostinfo_host_group' doesn't exist")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/hostsinfo/views.py", line 14, in get
    return render(request,"hosts-list.html",{'hosts':hosts,'groups':groups})
  File "/opt/python3.6/lib/python3.6/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/loader_tags.py", line 155, in render
    return compiled_parent._render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/loader_tags.py", line 67, in render
    result = block.nodelist.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/defaulttags.py", line 211, in render
    nodelist.append(node.render_annotated(context))
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/defaulttags.py", line 168, in render
    len_values = len(values)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/query.py", line 254, in __len__
    self._fetch_all()
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/query.py", line 1179, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/query.py", line 53, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1067, in execute_sql
    cursor.execute(sql, params)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/backends/utils.py", line 100, in execute
    return super().execute(sql, params)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/backends/utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/backends/utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/backends/mysql/base.py", line 71, in execute
    return self.cursor.execute(query, args)
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/cursors.py", line 255, in execute
    self.errorhandler(self, exc, value)
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/connections.py", line 50, in defaulterrorhandler
    raise errorvalue
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/cursors.py", line 252, in execute
    res = self._query(query)
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/cursors.py", line 378, in _query
    db.query(q)
  File "/opt/python3.6/lib/python3.6/site-packages/MySQLdb/connections.py", line 280, in query
    _mysql.connection.query(self, query)
django.db.utils.ProgrammingError: (1146, "Table 'opsauto.hostinfo_host_group' doesn't exist")
[E 20190506 22:34:48 basehttp:124] "GET /hostinfo/ HTTP/1.1" 500 280944
[I 20190506 22:39:32 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 11772
[I 20190506 22:39:32 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[I 20190506 22:39:34 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 10442
[I 20190506 22:39:35 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190506 22:39:52 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 10442
[I 20190506 22:39:53 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190506 22:40:11 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 10442
[I 20190506 22:42:13 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190506 22:42:15 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 12817
[I 20190506 22:42:16 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190506 22:42:32 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 13994
[I 20190506 22:42:41 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 12249
[I 20190506 22:42:43 basehttp:124] "GET /addhost/ HTTP/1.1" 200 14477
[I 20190506 22:42:55 basehttp:124] "POST /collecthost/ HTTP/1.1" 200 14619
[I 20190506 22:42:59 basehttp:124] "POST /addhost/ HTTP/1.1" 200 13967
[I 20190506 22:43:02 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 13643
[I 20190506 22:43:33 basehttp:124] "GET /addhost/ HTTP/1.1" 200 14477
[I 20190506 22:44:37 basehttp:124] "POST /collecthost/ HTTP/1.1" 200 14619
[I 20190506 22:44:39 basehttp:124] "POST /addhost/ HTTP/1.1" 200 13967
[I 20190506 22:45:21 basehttp:124] "POST /collecthost/ HTTP/1.1" 200 14619
[I 20190506 22:45:27 basehttp:124] "POST /addhost/ HTTP/1.1" 200 978
[W 20190506 22:45:27 base:93] Not Found: /addhost/vendor/simple-line-icons/css/simple-line-icons.css
[W 20190506 22:45:27 basehttp:124] "GET /addhost/vendor/simple-line-icons/css/simple-line-icons.css HTTP/1.1" 404 3773
[W 20190506 22:45:27 base:93] Not Found: /addhost/vendor/font-awesome/css/fontawesome-all.min.css
[W 20190506 22:45:27 basehttp:124] "GET /addhost/vendor/font-awesome/css/fontawesome-all.min.css HTTP/1.1" 404 3764
[W 20190506 22:45:27 base:93] Not Found: /addhost/css/styles.css
[W 20190506 22:45:27 basehttp:124] "GET /addhost/css/styles.css HTTP/1.1" 404 3665
[E 20190506 22:45:44 exception:118] Internal Server Error: /collecthost/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/hostsinfo/views.py", line 89, in post
    result = nm.collection(host_ip,host_password)
  File "/root/OpsAuto/hostsinfo/utils.py", line 137, in collection
    jssh.connect(hostname=ip, port=22, username='root', password=password)
  File "/opt/python3.6/lib/python3.6/site-packages/paramiko/client.py", line 310, in connect
    retry_on_signal(lambda: sock.connect(addr))
  File "/opt/python3.6/lib/python3.6/site-packages/paramiko/util.py", line 276, in retry_on_signal
    return function()
  File "/opt/python3.6/lib/python3.6/site-packages/paramiko/client.py", line 310, in <lambda>
    retry_on_signal(lambda: sock.connect(addr))
TimeoutError: [Errno 110] Connection timed out
[E 20190506 22:45:44 basehttp:124] "POST /collecthost/ HTTP/1.1" 500 93819
[I 20190506 22:46:07 basehttp:124] "POST /addhost/ HTTP/1.1" 200 996
[E 20190506 22:47:06 exception:118] Internal Server Error: /collecthost/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/hostsinfo/views.py", line 89, in post
    result = nm.collection(host_ip,host_password)
  File "/root/OpsAuto/hostsinfo/utils.py", line 137, in collection
    jssh.connect(hostname=ip, port=22, username='root', password=password)
  File "/opt/python3.6/lib/python3.6/site-packages/paramiko/client.py", line 310, in connect
    retry_on_signal(lambda: sock.connect(addr))
  File "/opt/python3.6/lib/python3.6/site-packages/paramiko/util.py", line 276, in retry_on_signal
    return function()
  File "/opt/python3.6/lib/python3.6/site-packages/paramiko/client.py", line 310, in <lambda>
    retry_on_signal(lambda: sock.connect(addr))
TimeoutError: [Errno 110] Connection timed out
[E 20190506 22:47:07 basehttp:124] "POST /collecthost/ HTTP/1.1" 500 93823
[I 20190506 22:47:40 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10971
[I 20190506 22:47:40 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[I 20190506 22:47:42 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11781
[I 20190506 22:47:50 basehttp:124] "POST /killttyp/ HTTP/1.1" 200 12708
[I 20190506 22:49:18 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10971
[I 20190506 22:49:58 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 15018
[I 20190506 22:50:06 basehttp:124] "POST /hostinfo/ HTTP/1.1" 200 15018
[I 20190506 22:50:09 basehttp:124] "POST /hostinfo/ HTTP/1.1" 200 13643
[I 20190506 22:50:26 basehttp:124] "POST /hostinfo/ HTTP/1.1" 200 15018
[I 20190506 22:52:13 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11886
[I 20190506 22:52:17 basehttp:124] "POST /killttyp/ HTTP/1.1" 200 12813
[I 20190506 22:52:55 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11783
[I 20190506 22:52:57 basehttp:124] "POST /killttyp/ HTTP/1.1" 200 12816
[I 20190506 22:55:01 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11783
[I 20190506 22:55:01 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[I 20190506 22:55:04 basehttp:124] "POST /killttyp/ HTTP/1.1" 200 12837
[I 20190506 23:00:39 basehttp:124] "GET /upload/ HTTP/1.1" 200 11289
[I 20190506 23:01:13 basehttp:124] "GET /index/ HTTP/1.1" 200 13311
[I 20190506 23:01:15 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 15018
[I 20190506 23:01:18 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 13994
[I 20190506 23:01:37 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 15018
[I 20190506 23:01:40 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11783
[I 20190506 23:01:46 basehttp:124] "GET /upload/ HTTP/1.1" 200 11289
[W 20190506 23:01:49 base:93] Not Found: /upload/chartjs.html
[W 20190506 23:01:49 basehttp:124] "GET /upload/chartjs.html HTTP/1.1" 404 3656
[I 20190507 10:33:58 basehttp:124] "GET /index/ HTTP/1.1" 200 13311
[W 20190507 10:33:59 base:93] Not Found: /favicon.ico
[W 20190507 10:33:59 basehttp:124] "GET /favicon.ico HTTP/1.1" 404 3632
[I 20190507 10:34:03 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10949
[I 20190507 10:35:37 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10971
[W 20190507 10:37:43 base:93] Not Found: /userlist/task-do.html
[W 20190507 10:37:43 basehttp:124] "GET /userlist/task-do.html HTTP/1.1" 404 3662
[I 20190507 10:37:46 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11783
[I 20190507 10:38:00 basehttp:124] "POST /killttyp/ HTTP/1.1" 200 12017
[I 20190507 10:43:37 basehttp:124] "POST /killttyp/ HTTP/1.1" 200 12422
[I 20190507 10:44:14 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 15121
[I 20190507 10:44:15 basehttp:124] "GET /addhost/ HTTP/1.1" 200 14619
[I 20190507 10:44:17 basehttp:124] "POST /collecthost/ HTTP/1.1" 200 14725
[I 20190507 10:44:24 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 15121
[I 20190507 10:44:28 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 15121
[I 20190507 10:47:07 basehttp:124] "GET /addhost/ HTTP/1.1" 200 14619
[I 20190507 10:47:10 basehttp:124] "POST /collecthost/ HTTP/1.1" 200 14751
[I 20190507 10:48:19 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 15139
[I 20190507 10:48:47 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 15139
[I 20190507 10:48:48 basehttp:124] "GET /addhost/ HTTP/1.1" 200 14619
[I 20190507 10:48:51 basehttp:124] "POST /collecthost/ HTTP/1.1" 200 14751
[I 20190507 10:49:28 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 15139
[I 20190507 10:49:32 basehttp:124] "GET /hostdetail2/ HTTP/1.1" 200 10992
[W 20190507 10:49:41 base:93] Not Found: /hostinfo/host-detail.html
[W 20190507 10:49:41 basehttp:124] "GET /hostinfo/host-detail.html HTTP/1.1" 404 3674
[W 20190507 10:49:43 base:93] Not Found: /hostinfo/host-detail.html
[W 20190507 10:49:43 basehttp:124] "GET /hostinfo/host-detail.html HTTP/1.1" 404 3674
[I 20190507 11:01:53 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 15127
[I 20190507 11:01:59 basehttp:124] "GET /hostdel2/ HTTP/1.1" 200 13776
[I 20190507 11:02:03 basehttp:124] "GET /addhost/ HTTP/1.1" 200 14619
[I 20190507 11:02:06 basehttp:124] "POST /collecthost/ HTTP/1.1" 200 14751
[I 20190507 11:02:15 basehttp:124] "POST /addhost/ HTTP/1.1" 200 13967
[I 20190507 11:02:17 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 15078
[I 20190507 11:02:21 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 15102
[I 20190507 11:02:25 basehttp:124] "GET /hostdel3/ HTTP/1.1" 200 13776
[I 20190507 11:02:27 basehttp:124] "GET /addhost/ HTTP/1.1" 200 14619
[I 20190507 11:02:30 basehttp:124] "POST /collecthost/ HTTP/1.1" 200 14751
[I 20190507 11:02:38 basehttp:124] "POST /addhost/ HTTP/1.1" 200 13967
[I 20190507 11:02:40 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 15127
[I 20190507 11:29:08 basehttp:124] "GET /usersettings/ HTTP/1.1" 200 13906
[W 20190507 11:30:38 basehttp:124] "GET /static/vendor/bootstrap/js/bootstrap.min.js.map HTTP/1.1" 404 1744
[I 20190507 11:37:35 basehttp:124] "GET /index/ HTTP/1.1" 200 13311
[W 20190507 11:37:35 basehttp:124] "GET /static/vendor/bootstrap/js/bootstrap.min.js.map HTTP/1.1" 404 1744
[I 20190507 11:37:41 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10997
[E 20190507 11:37:42 exception:118] Internal Server Error: /userprofile1/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
TypeError: get() got an unexpected keyword argument 'host_id'
[E 20190507 11:37:42 basehttp:124] "GET /userprofile1/ HTTP/1.1" 500 69323
[I 20190507 11:38:56 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10997
[E 20190507 11:39:02 exception:118] Internal Server Error: /userprofile1/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
TypeError: get() got an unexpected keyword argument 'host_id'
[E 20190507 11:39:02 basehttp:124] "GET /userprofile1/ HTTP/1.1" 500 69323
[E 20190507 11:40:21 exception:118] Internal Server Error: /userprofile1/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
TypeError: get() got an unexpected keyword argument 'host_id'
[E 20190507 11:40:21 basehttp:124] "GET /userprofile1/ HTTP/1.1" 500 69323
[I 20190507 11:40:43 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10997
[I 20190507 11:40:44 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12612
[I 20190507 12:08:07 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12583
[I 20190507 12:08:59 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 10474
[I 20190507 12:09:32 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 10715
[I 20190507 12:09:48 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 10884
[I 20190507 12:10:05 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 10712
[I 20190507 12:10:28 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 10712
[I 20190507 12:10:31 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12583
[I 20190507 12:10:46 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12153
[I 20190507 12:11:32 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12173
[I 20190507 12:11:36 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10997
[I 20190507 12:11:37 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12173
[I 20190507 12:12:09 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10997
[I 20190507 12:12:11 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12206
[I 20190507 12:13:05 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12349
[I 20190507 12:17:37 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12352
[I 20190507 12:19:10 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12367
[I 20190507 12:20:49 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12383
[I 20190507 12:21:32 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12408
[I 20190507 12:22:18 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12450
[I 20190507 12:24:44 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12423
[I 20190507 12:25:05 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12423
[I 20190507 12:25:29 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12425
[I 20190507 12:26:06 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12466
[I 20190507 12:27:15 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12456
[I 20190507 12:27:47 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12469
[I 20190507 12:28:03 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12470
[I 20190507 12:28:49 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12446
[I 20190507 12:29:12 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12445
[I 20190507 12:29:30 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12443
[I 20190507 13:57:03 basehttp:124] "GET /index/ HTTP/1.1" 200 13311
[I 20190507 14:03:02 basehttp:124] "GET /index/ HTTP/1.1" 200 13309
[I 20190507 14:03:23 basehttp:124] "GET /index/ HTTP/1.1" 200 13309
[I 20190507 14:03:41 basehttp:124] "GET /index/ HTTP/1.1" 200 13305
[I 20190507 14:10:22 basehttp:124] "GET /index/ HTTP/1.1" 200 13305
[I 20190507 14:11:22 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 15078
[I 20190507 14:11:26 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 15127
[I 20190507 14:11:49 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 15078
[I 20190507 14:12:00 basehttp:124] "GET /index/ HTTP/1.1" 200 13488
[I 20190507 14:12:05 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 15127
[I 20190507 14:12:54 basehttp:124] "GET /index/ HTTP/1.1" 200 13495
[I 20190507 14:13:16 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 15127
[E 20190507 14:32:03 exception:118] Internal Server Error: /grouplist/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/hostsinfo/views.py", line 110, in get
    p = Paginator(groups,request=request)
TypeError: __init__() missing 1 required positional argument: 'per_page'
[E 20190507 14:32:04 basehttp:124] "GET /grouplist/ HTTP/1.1" 500 73608
[E 20190507 14:34:11 exception:118] Internal Server Error: /grouplist/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/hostsinfo/views.py", line 113, in get
    return render(request,"group-list.html",{'groups':groups})
  File "/opt/python3.6/lib/python3.6/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/loader_tags.py", line 155, in render
    return compiled_parent._render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/loader_tags.py", line 67, in render
    result = block.nodelist.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/opt/python3.6/lib/python3.6/site-packages/django/template/defaulttags.py", line 167, in render
    values = list(values)
TypeError: 'Page' object is not iterable
[E 20190507 14:34:11 basehttp:124] "GET /grouplist/ HTTP/1.1" 500 158911
[I 20190507 14:39:26 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 15446
[I 20190507 14:45:48 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 14407
[I 20190507 14:45:55 basehttp:124] "GET /grouplist/?page=2 HTTP/1.1" 200 12065
[I 20190507 14:45:57 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 14407
[I 20190507 14:46:54 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 14576
[I 20190507 14:47:38 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 14576
[I 20190507 14:48:37 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 14177
[I 20190507 14:48:39 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 14177
[I 20190507 14:49:22 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 14097
[I 20190507 14:49:40 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 14085
[I 20190507 14:50:07 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 14100
[I 20190507 14:50:09 basehttp:124] "GET /grouplist/?page=2 HTTP/1.1" 200 11758
[I 20190507 14:50:12 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 14100
[I 20190507 14:51:00 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 14121
[W 20190507 14:51:11 basehttp:124] "GET /static/vendor/bootstrap/js/bootstrap.min.js.map HTTP/1.1" 404 1744
[I 20190507 14:52:10 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 14126
[I 20190507 14:52:12 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 14:52:20 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 10481
[I 20190507 14:52:21 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 14:53:05 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 10481
[I 20190507 14:53:56 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 14:54:04 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 10481
[I 20190507 14:54:05 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 14:54:12 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 10481
[I 20190507 14:54:55 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 14:55:00 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 10481
[I 20190507 14:55:58 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 14:56:09 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 14126
[I 20190507 14:56:11 basehttp:124] "GET /addgroup/?page=2 HTTP/1.1" 200 11094
[I 20190507 14:56:17 basehttp:124] "GET /index/ HTTP/1.1" 200 13496
[I 20190507 14:56:18 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 16105
[I 20190507 14:56:20 basehttp:124] "GET /index/ HTTP/1.1" 200 13496
[I 20190507 14:56:21 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 14126
[I 20190507 14:56:23 basehttp:124] "GET /grouplist/?page=2 HTTP/1.1" 200 14303
[I 20190507 14:56:27 basehttp:124] "GET /grouplist/?page=3 HTTP/1.1" 200 14351
[I 20190507 14:56:28 basehttp:124] "GET /grouplist/?page=4 HTTP/1.1" 200 11832
[I 20190507 14:56:29 basehttp:124] "GET /grouplist/?page=3 HTTP/1.1" 200 14351
[I 20190507 14:56:31 basehttp:124] "GET /grouplist/?page=2 HTTP/1.1" 200 14303
[I 20190507 14:57:30 basehttp:124] "GET /grouplist/?page=2 HTTP/1.1" 200 14288
[I 20190507 14:58:08 basehttp:124] "GET /grouplist/?page=2 HTTP/1.1" 200 15696
[I 20190507 14:58:18 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 15519
[I 20190507 14:59:55 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 15479
[I 20190507 15:09:07 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 15479
[I 20190507 15:09:11 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 15503
[I 20190507 15:09:26 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 15487
[I 20190507 15:09:39 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 15471
[I 20190507 15:09:55 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 15341
[I 20190507 15:10:13 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 15341
[I 20190507 15:10:30 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 15351
[I 20190507 15:15:50 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 15265
[I 20190507 15:16:05 basehttp:124] "GET /grouplist/?page=2 HTTP/1.1" 200 15419
[I 20190507 15:16:07 basehttp:124] "GET /grouplist/?page=3 HTTP/1.1" 200 15467
[I 20190507 15:16:08 basehttp:124] "GET /grouplist/?page=4 HTTP/1.1" 200 12970
[I 20190507 15:16:09 basehttp:124] "GET /grouplist/?page=3 HTTP/1.1" 200 15467
[I 20190507 15:16:10 basehttp:124] "GET /grouplist/?page=2 HTTP/1.1" 200 15419
[I 20190507 15:17:01 basehttp:124] "GET /grouplist/?page=2 HTTP/1.1" 200 15434
[I 20190507 15:17:09 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 15280
[I 20190507 15:17:20 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 15282
[I 20190507 15:17:30 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 15297
[I 20190507 15:17:40 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 15296
[I 20190507 15:18:10 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 22382
[I 20190507 15:18:25 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 15:18:33 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 22866
[I 20190507 15:18:38 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 15:18:42 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 22866
[I 20190507 15:18:53 basehttp:124] "GET /index/ HTTP/1.1" 200 13496
[I 20190507 15:18:56 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 22866
[I 20190507 15:19:01 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 16435
[I 20190507 15:19:03 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10997
[I 20190507 15:19:05 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12443
[I 20190507 15:27:51 basehttp:124] "GET /usersettings/ HTTP/1.1" 200 13906
[I 20190507 15:27:56 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 16435
[I 20190507 21:08:21 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11783
[I 20190507 21:08:21 basehttp:124] "GET /static/imgs/logo.png HTTP/1.1" 304 0
[W 20190507 21:08:21 base:93] Not Found: /favicon.ico
[W 20190507 21:08:21 basehttp:124] "GET /favicon.ico HTTP/1.1" 404 3934
[I 20190507 21:08:24 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 14182
[I 20190507 21:08:35 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 21:08:44 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 15335
[I 20190507 21:08:45 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 21:08:47 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 997
[I 20190507 21:08:49 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 21:08:54 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 16488
[I 20190507 21:08:55 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 21:09:00 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 17641
[I 20190507 21:09:02 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 21:09:08 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 18794
[I 20190507 21:09:09 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 21:09:15 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 19947
[I 20190507 21:09:16 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 21:09:21 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 21100
[I 20190507 21:09:22 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 21:09:27 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 22253
[I 20190507 21:09:28 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 21:09:33 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 22737
[I 20190507 21:09:34 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 21:09:39 basehttp:124] "POST /addgroup/ HTTP/1.1" 200 22737
[I 20190507 21:10:07 basehttp:124] "GET /addgroup/ HTTP/1.1" 200 11094
[I 20190507 21:10:15 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 22737
[I 20190507 21:10:20 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 22737
[I 20190507 21:10:22 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 22737
[I 20190507 21:10:22 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 22737
[I 20190507 21:10:24 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 16275
[I 20190507 21:10:25 basehttp:124] "GET /hostdetail1/ HTTP/1.1" 200 11030
[I 20190507 21:10:32 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 22737
[I 20190507 21:10:34 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 16275
[I 20190507 21:10:45 basehttp:124] "GET /index/ HTTP/1.1" 200 13496
[I 20190507 21:10:57 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11783
[I 20190507 21:11:18 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10997
[I 20190507 21:11:20 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12436
[I 20190507 21:11:33 basehttp:124] "GET /index/ HTTP/1.1" 200 13496
[I 20190507 21:11:35 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 16275
[I 20190507 21:15:27 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 22760
[I 20190507 21:16:11 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 13779
[I 20190507 21:16:14 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 13779
[I 20190507 21:16:52 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 13779
[I 20190507 21:16:54 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 13779
[I 20190507 21:17:58 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 13779
[I 20190507 21:17:59 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 13779
[I 20190507 21:18:33 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 13779
[I 20190507 21:18:59 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 16864
[I 20190507 21:19:32 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 16864
[I 20190507 21:19:34 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 22760
[I 20190507 21:19:39 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 16864
[I 20190507 21:24:31 basehttp:124] "GET /usersettings/ HTTP/1.1" 200 13906
[I 20190507 21:24:59 basehttp:124] "GET /index/ HTTP/1.1" 200 13496
[I 20190507 21:25:03 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 16864
[I 20190507 21:31:39 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 22760
[I 20190507 21:31:50 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11783
[I 20190507 21:32:01 basehttp:124] "GET /upload/ HTTP/1.1" 200 12549
[W 20190507 21:32:03 base:93] Not Found: /upload/tables.html
[W 20190507 21:32:03 basehttp:124] "GET /upload/tables.html HTTP/1.1" 404 3955
[I 20190507 21:32:21 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11783
[I 20190507 21:32:23 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 16864
[I 20190507 21:32:31 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 16864
[W 20190507 21:33:01 base:93] Not Found: /hostinfo/task-do.html
[W 20190507 21:33:01 basehttp:124] "GET /hostinfo/task-do.html HTTP/1.1" 404 3964
[W 20190507 21:33:03 base:93] Not Found: /hostinfo/chartjs.html
[W 20190507 21:33:03 basehttp:124] "GET /hostinfo/chartjs.html HTTP/1.1" 404 3964
[I 20190509 14:08:22 basehttp:124] "GET /index/ HTTP/1.1" 200 13496
[W 20190509 14:08:23 base:93] Not Found: /favicon.ico
[W 20190509 14:08:23 basehttp:124] "GET /favicon.ico HTTP/1.1" 404 3934
[I 20190509 14:08:25 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 17024
[I 20190509 14:08:28 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 22889
[I 20190509 14:08:31 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10997
[I 20190509 14:08:35 basehttp:124] "GET /upload/ HTTP/1.1" 200 12719
[I 20190509 14:08:36 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10997
[I 20190509 14:08:40 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11783
[I 20190509 14:11:34 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11442
[I 20190509 14:12:10 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11363
[I 20190509 14:14:12 basehttp:124] "POST /killttyp/ HTTP/1.1" 200 11554
[I 20190509 14:14:28 basehttp:124] "POST /killttyp/ HTTP/1.1" 200 12860
[I 20190509 14:16:12 basehttp:124] "POST /killttyp/ HTTP/1.1" 200 12965
[I 20190509 14:17:38 basehttp:124] "POST /killttyp/ HTTP/1.1" 200 12969
[I 20190509 14:18:00 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11488
[I 20190509 14:18:04 basehttp:124] "POST /killttyp/ HTTP/1.1" 200 13643
[I 20190509 14:18:31 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11492
[I 20190509 14:18:35 basehttp:124] "POST /killttyp/ HTTP/1.1" 200 13647
[I 20190509 14:24:40 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11507
[I 20190509 14:25:09 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11507
[I 20190509 14:25:27 basehttp:124] "POST /killttyp/ HTTP/1.1" 200 12984
[I 20190509 14:36:05 basehttp:124] "GET /index/ HTTP/1.1" 200 13496
[I 20190509 14:36:06 basehttp:124] "GET /upload/ HTTP/1.1" 200 13487
[E 20190509 14:36:41 exception:118] Internal Server Error: /upload/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/taskdo/views.py", line 106, in post
    stdout_success[host] = result['success'][host]['stdout']
KeyError: 'stdout'
[E 20190509 14:36:41 basehttp:124] "POST /upload/ HTTP/1.1" 500 88258
[E 20190509 14:37:25 exception:118] Internal Server Error: /upload/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/taskdo/views.py", line 106, in post
    stdout_success[host] = result['success'][host]['stdout']
KeyError: 'stdout'
[E 20190509 14:37:25 basehttp:124] "POST /upload/ HTTP/1.1" 500 88434
[I 20190509 14:39:10 basehttp:124] "POST /upload/ HTTP/1.1" 200 13642
[I 20190509 14:50:19 basehttp:124] "GET /upload/ HTTP/1.1" 200 13487
[E 20190509 14:50:39 exception:118] Internal Server Error: /upload/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/taskdo/views.py", line 108, in post
    stdout_success[host] = result['success'][host]['backupfile'] + result['success'][host]['changed']
KeyError: 'backupfile'
[E 20190509 14:50:40 basehttp:124] "POST /upload/ HTTP/1.1" 500 88570
[I 20190509 14:51:32 basehttp:124] "GET /upload/ HTTP/1.1" 200 13487
[I 20190509 14:51:52 basehttp:124] "POST /upload/ HTTP/1.1" 200 14031
[I 20190509 14:54:41 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 17024
[I 20190509 14:54:43 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 22889
[I 20190509 14:54:45 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 22889
[I 20190509 14:55:18 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 17024
[I 20190509 14:55:21 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 22889
[I 20190509 14:56:13 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 17024
[I 20190509 14:59:25 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 22889
[I 20190509 15:02:29 basehttp:124] "GET /index/ HTTP/1.1" 200 13496
[I 20190509 15:02:33 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 22889
[W 20190509 15:02:34 base:93] Not Found: /grouplist/host-detail.html
[W 20190509 15:02:34 basehttp:124] "GET /grouplist/host-detail.html HTTP/1.1" 404 4133
[I 20190509 15:03:02 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 22899
[W 20190509 15:03:03 base:93] Not Found: /grouplist/group-detail.html
[W 20190509 15:03:03 basehttp:124] "GET /grouplist/group-detail.html HTTP/1.1" 404 4136
[I 20190509 15:03:22 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 21299
[W 20190509 15:03:28 base:93] Not Found: /group-detail/"
[W 20190509 15:03:28 basehttp:124] "GET /group-detail/%22 HTTP/1.1" 404 4109
[I 20190509 15:03:48 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 20820
[W 20190509 15:03:49 base:93] Not Found: /group-detail1/"
[W 20190509 15:03:49 basehttp:124] "GET /group-detail1/%22 HTTP/1.1" 404 4112
[I 20190509 15:04:11 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 20810
[I 20190509 15:04:12 basehttp:124] "GET /modifygroup1/%22 HTTP/1.1" 301 0
[E 20190509 15:04:12 exception:118] Internal Server Error: /modifygroup1/"/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/hostsinfo/views.py", line 194, in get
    group = HostGroup.objects.get(id=host_id)
NameError: name 'host_id' is not defined
[E 20190509 15:04:13 basehttp:124] "GET /modifygroup1/%22/ HTTP/1.1" 500 73011
[I 20190509 15:04:32 basehttp:124] "GET /modifygroup2/%22 HTTP/1.1" 301 0
[E 20190509 15:04:32 exception:118] Internal Server Error: /modifygroup2/"/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/hostsinfo/views.py", line 194, in get
    group = HostGroup.objects.get(id=host_id)
NameError: name 'host_id' is not defined
[E 20190509 15:04:32 basehttp:124] "GET /modifygroup2/%22/ HTTP/1.1" 500 73011
[E 20190509 15:04:51 exception:118] Internal Server Error: /modifygroup1/"/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/hostsinfo/views.py", line 194, in get
    group = HostGroup.objects.get(id=group_id)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/query.py", line 394, in get
    clone = self.filter(*args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/query.py", line 836, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/query.py", line 854, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1253, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1277, in _add_q
    split_subq=split_subq,
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1215, in build_filter
    condition = self.build_lookup(lookups, col, value)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1085, in build_lookup
    lookup = lookup_class(lhs, rhs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/lookups.py", line 18, in __init__
    self.rhs = self.get_prep_lookup()
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/lookups.py", line 68, in get_prep_lookup
    return self.lhs.output_field.get_prep_value(self.rhs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/db/models/fields/__init__.py", line 947, in get_prep_value
    return int(value)
ValueError: invalid literal for int() with base 10: '1/"'
[E 20190509 15:04:51 basehttp:124] "GET /modifygroup1/%22/ HTTP/1.1" 500 126206
[E 20190509 15:06:36 exception:118] Internal Server Error: /modifygroup1/"/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/hostsinfo/views.py", line 194, in get
    group = HostGroup.objects.get(id=int(group_id))
ValueError: invalid literal for int() with base 10: '1/"'
[E 20190509 15:06:36 basehttp:124] "GET /modifygroup1/%22/ HTTP/1.1" 500 73113
[I 20190509 15:06:43 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 20810
[E 20190509 15:06:45 exception:118] Internal Server Error: /modifygroup1/"/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/hostsinfo/views.py", line 194, in get
    group = HostGroup.objects.get(id=int(group_id))
ValueError: invalid literal for int() with base 10: '1/"'
[E 20190509 15:06:45 basehttp:124] "GET /modifygroup1/%22/ HTTP/1.1" 500 73113
[I 20190509 15:08:00 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 20810
[E 20190509 15:08:01 exception:118] Internal Server Error: /modifygroup1/"/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/hostsinfo/views.py", line 194, in get
    group = HostGroup.objects.get(id=int(group_id))
ValueError: invalid literal for int() with base 10: '1/"'
[E 20190509 15:08:02 basehttp:124] "GET /modifygroup1/%22/ HTTP/1.1" 500 73378
[I 20190509 15:08:25 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 20800
[E 20190509 15:08:26 exception:118] Internal Server Error: /modifygroup1/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/hostsinfo/views.py", line 198, in get
    return render(request,"group-detail.html",{'group':group})
NameError: name 'request' is not defined
[E 20190509 15:08:26 basehttp:124] "GET /modifygroup1/ HTTP/1.1" 500 72598
[E 20190509 15:08:56 exception:118] Internal Server Error: /modifygroup1/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/hostsinfo/views.py", line 198, in get
    return render(request,"group-detail.html",{'group':group})
NameError: name 'request' is not defined
[E 20190509 15:08:56 basehttp:124] "GET /modifygroup1/ HTTP/1.1" 500 72598
[I 20190509 15:09:09 basehttp:124] "GET /modifygroup1/ HTTP/1.1" 200 11161
[I 20190509 15:09:14 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 20800
[I 20190509 15:09:15 basehttp:124] "GET /modifygroup1/ HTTP/1.1" 200 11161
[I 20190509 15:10:45 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 20800
[I 20190509 15:10:46 basehttp:124] "GET /modifygroup1/ HTTP/1.1" 200 11198
[I 20190509 15:11:23 basehttp:124] "GET /modifygroup1/ HTTP/1.1" 200 11169
[I 20190509 15:11:43 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 20800
[I 20190509 15:11:47 basehttp:124] "GET /modifygroup2/ HTTP/1.1" 200 11184
[I 20190509 15:25:43 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 20751
[I 20190509 15:25:46 basehttp:124] "GET /delgroup4/ HTTP/1.1" 302 0
[I 20190509 15:25:46 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 20804
[I 20190509 15:25:49 basehttp:124] "GET /grouplist/?page=2 HTTP/1.1" 200 12114
[I 20190509 15:25:51 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 20804
[I 20190509 15:25:54 basehttp:124] "GET /modifygroup1/ HTTP/1.1" 200 11169
[E 20190509 15:26:03 exception:118] Internal Server Error: /modifygroup/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/hostsinfo/views.py", line 202, in post
    HostGroup.objects.filter(id=int(group_id)).update(group_name=group_name,network=group_network,group_detail=group_detail)
ValueError: invalid literal for int() with base 10: ''
[E 20190509 15:26:03 basehttp:124] "POST /modifygroup/ HTTP/1.1" 500 75438
[E 20190509 15:27:46 exception:118] Internal Server Error: /modifygroup/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/root/OpsAuto/hostsinfo/views.py", line 204, in post
    HostGroup.objects.filter(id=int(group_id)).update(group_name=group_name,network=group_network,group_detail=group_detail)
ValueError: invalid literal for int() with base 10: ''
[E 20190509 15:27:46 basehttp:124] "POST /modifygroup/ HTTP/1.1" 500 75359
[I 20190509 15:30:03 basehttp:124] "GET /grouplist/?page=1 HTTP/1.1" 200 20804
[I 20190509 15:30:05 basehttp:124] "GET /modifygroup1/ HTTP/1.1" 200 11265
[E 20190509 15:30:13 exception:118] Internal Server Error: /modifygroup/
Traceback (most recent call last):
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/python3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/opt/python3.6/lib/python3.6/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
TypeError: post() got an unexpected keyword argument 'group_id'
[E 20190509 15:30:13 basehttp:124] "POST /modifygroup/ HTTP/1.1" 500 70882
[I 20190509 15:30:57 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 20804
[I 20190509 15:30:58 basehttp:124] "GET /modifygroup1/ HTTP/1.1" 200 11265
[I 20190509 15:31:03 basehttp:124] "POST /modifygroup/ HTTP/1.1" 302 0
[I 20190509 15:31:03 basehttp:124] "GET /grouplist/ HTTP/1.1" 200 20831
[I 20190509 15:45:47 basehttp:124] "GET /hostinfo/ HTTP/1.1" 200 16893
[I 20190509 15:45:49 basehttp:124] "GET /hostdetail1/ HTTP/1.1" 200 11030
[W 20190509 15:45:53 base:93] Not Found: /hostinfo/host-detail.html
[W 20190509 15:45:53 basehttp:124] "GET /hostinfo/host-detail.html HTTP/1.1" 404 4280
[I 20190509 15:47:16 basehttp:124] "GET /killttyp/ HTTP/1.1" 200 11509
[I 20190509 15:47:18 basehttp:124] "GET /upload/ HTTP/1.1" 200 13357
[I 20190509 15:47:24 basehttp:124] "GET /userlist/ HTTP/1.1" 200 10997
[I 20190509 15:47:26 basehttp:124] "GET /userprofile1/ HTTP/1.1" 200 12443
